exchanges/orderbook/orderbook.go

```diff
@@ -206,62 +184,114 @@ func (b *Base) Update(bids, asks []Item) {
 	b.LastUpdated = time.Now()
 }
 
-// Verify ensures that the orderbook items are correctly sorted
+// Verify ensures that the orderbook items are correctly sorted prior to being
+// set and will reject any book with incorrect values.
 // Bids should always go from a high price to a low price and
-// asks should always go from a low price to a higher price
-func (b *Base) Verify() {
-	var lastPrice float64
-	var sortBids, sortAsks bool
-	for x := range b.Bids {
-		if lastPrice != 0 && b.Bids[x].Price >= lastPrice {
-			sortBids = true
-			break
-		}
-		lastPrice = b.Bids[x].Price
+// Asks should always go from a low price to a higher price
+func (b *Base) Verify() error {
```

Comment:
So when this returns an error, it gets passed back to the exchange's websocket to highlight that the orderbook isn't valid anymore. Any further updates to it would make it out of sync, right? What should we do? Unsubscribe from that orderbook? Resubscribe? With `syncer.go` at least we'll know that we can fallback to REST, so resubscribing isn't paramount.

Reply:
Resubscribe might be the best option. Focus on that for another PR. There was an issue in Kraken that I need to inspect. As it was locking the entire reader routine.  [QuickFixHere](https://github.com/thrasher-corp/gocryptotrader/pull/599/commits/685587c13c4ffa2aea193aaf1e192293f9eb2d19#diff-0c0a7ee0ec6a7ec64a0d50fd778775c822378d3c5c87b775c7373d658d7771bcR752)